doc-content/enterprise-only/openshift/jboss-postconfigure-proc.adoc

```diff
@@ -0,0 +1,92 @@
+[id="jboss-postconfigure-proc_{context}"]
+= (Optional) Providing Elytron user configuration or other post-configuration settings
+
+If you do not use LDAP or RH-SSO authentication, you might need to add users to the Elytron security subsystem of {EAP}. To do so, you must run an {EAP} post-configuration script.
```

Comment:
It might not work with ldap and sso, it is just to add extra users than the one created witht he envs KIE_ADMIN_USERNAME and PASSWORD.